src/mx/uaemex/fi/paradigmas_I/figuras/vista/PanelCirculoData.java

```diff
@@ -0,0 +1,45 @@
+package mx.uaemex.fi.paradigmas_I.figuras.vista;
+
+import mx.uaemex.fi.paradigmas_I.figuras.data.Figura;
+
+import javax.swing.*;
+import javax.swing.border.EmptyBorder;
+import java.awt.*;
+
+public class PanelCirculoData extends PanelDeFigura{
+
+    private JSpinner radio;
+
+    public PanelCirculoData(){
+        super();
+
+        JLabel etqRadio;
+        SpinnerModel modelo;
+        JButton btnGD;
+
+        setLayout(new GridLayout(3,1));
```

Comment:
Cómo sobra el botón y para mantener estilos, es mejor hacer lo siguiente
```suggestion
        setLayout(new GridLayout(1,2));
```
De esta forma hay dos columnas, una para el label y otra para el spinner